gatsby-browser.js

```diff
@@ -29,13 +29,14 @@ import FeatureFlagProvider from "decentraland-gatsby/dist/context/FeatureFlag/Fe
 import ShareProvider from "decentraland-gatsby/dist/context/Share/ShareProvider"
 import { IntlProvider } from "decentraland-gatsby/dist/plugins/intl"
 import segment from "decentraland-gatsby/dist/utils/development/segment"
+import env from "decentraland-gatsby/dist/utils/env"
 
 import UserMenu from "./src/components/Layout/UserMenu"
 
 export const registerServiceWorker = () => true
 
 export const wrapRootElement = ({ element }) => (
-  <AuthProvider>
+  <AuthProvider sso={env("SSO_URL")}>
```

Comment:
Just in case, best way to do this is to add before something like 
`const ssoUrl = env(`SSO_URL `, `https://id.decentraland.org`)`

Reply:
Is there a file for constants I could add this?

Reply:
the ones you are using are ok. Mayeb we need to work in that second param (in the future)